ONSdigital/census-rm-docker-dev
# Update census rm ops image source, format file

# Motivation and Context
The census-rm-ops tool is moving to our census-rm-ci/rm registry with all our other images.

# What has changed
- Update census rm ops tool image source

# How to test?
Make up and navigate to the ops tool in browser it should now be our version with the census sample loader and should not display the baking in business surveys from the survey service.
NB: This is won't work until this PR goes in and a successful build/push from master goes through: https://github.com/ONSdigital/census-rm-ops/pull/1

# Links
https://trello.com/c/Ff1ZKDwP/584-fork-and-deploy-census-rm-ops-tool#
**Depends on** https://github.com/ONSdigital/census-rm-ops/pull/1
